force-app/main/default/classes/AccountSelectorTest.cls

```diff
@@ -0,0 +1,62 @@
+@isTest
+private class AccountSelectorTest {
+    @isTest
+    static void testGetAccountsByIds() {
+        // given
+        List<Account> accounts = new List<Account>{
+            new Account(Name='Test Account 1'),
+            new Account(Name='Test Account 2')
+        };
+        insert accounts;
+        
+        List<Contact> contacts = new List<Contact>{
+            new Contact(AccountId=accounts[0].Id, LastName='Test Contact 1'),
+            new Contact(AccountId=accounts[1].Id, LastName='Test Contact 2'),
+            new Contact(AccountId=accounts[1].Id, LastName='Test Contact 3')
+        };
+        insert contacts;
+        
+        // when
+        Set<Id> accountIds = new Set<Id>{accounts[0].Id, accounts[1].Id};
+        Map<Id, Account> result = AccountSelector.getAccountsByIds(accountIds);
+        
+        // then
+        System.assertEquals(2, result.size());
```

Comment:
Same here, messages is better 
And also check the new assert class 
https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_class_System_Assert.htm